cloudprober/templates/deployment.yaml

```diff
@@ -48,10 +48,19 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          args: ['--config_file', '/cfg/cloudprober.{{- .Values.configFormat -}}', '--logtostderr']
+          args:
+            - "--logtostderr"          
```

Comment:
We can drop this arg now. It doesn't do anything.

Reply:
Done